graphql/schemabuilder/reflect.go

```diff
@@ -56,6 +56,18 @@ func makeGraphql(s string) string {
 	return b.String()
 }
 
+func reverseGraphql(s string) string {
```

Comment:
this feels too vague as-is (as is `makeGraphql`..)

`reverseGraphqlFieldName`? `graphqlToGoName`?